crates/sui-types/src/messages.rs

```diff
@@ -62,6 +62,23 @@ pub const DUMMY_GAS_PRICE: u64 = 1;
 
 const BLOCKED_MOVE_FUNCTIONS: [(ObjectID, &str, &str); 0] = [];
 
+// Since `std::mem::size_of` may not be stable acrpss platforms, we use rough constans
+// We need these for estimating effects sizes
```

Comment:
```suggestion
// Since `std::mem::size_of` may not be stable across platforms, we use rough constants
// We need these for estimating effects sizes
```

Reply:
And `size_of` is only not stable across platforms if you use `usize`, right? All of these items already have well defined sizes because they already don't use usize for that issue.

Reply:
> And size_of is only not stable across platforms if you use usize, right? All of these items already have well defined sizes because they already don't use usize for that issue.

Not sure of the intricacies but didn't want to take the chance. Either way we have a test which will ensure we're overestimating the sizes at least.